Description: Fix python-based tests
 Remove $<CONFIGURATION> from the build dir supplied to python-based tests.
Author: Steve M. Robbins <smr@debian.org>
Forwarded: no
Last-Update: 2016-09-07

--- alioth.orig/googletest/cmake/internal_utils.cmake
+++ alioth/googletest/cmake/internal_utils.cmake
@@ -243,7 +243,7 @@
       add_test(
         NAME ${name}
         COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test/${name}.py
-            --build_dir=${CMAKE_CURRENT_BINARY_DIR}/$<CONFIGURATION>)
+            --build_dir=${CMAKE_CURRENT_BINARY_DIR})
     else (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} GREATER 3.1)
       add_test(
         ${name}
